www/blog/2023-01-08-tinyrpc-client.mdx

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
If you got time I'd love your flair on this @anthonyshew

Reply:
We should make it so our OG images can handle several authors 🤔